tests/unit/services/traction/Reception.spec.js

```diff
@@ -34,27 +34,31 @@ const createdReceptionResponse = {
   data: {},
 }
 
+const reception = {
```

Comment:
Could import this from src/lib/receptions/index.js

Reply:
Could do but I usually prefer to inline it if it is simple.